libs/labelbox/src/labelbox/data/serialization/coco/annotation.py

```diff
@@ -39,8 +35,8 @@ def get_annotation_lookup(
     annotation_lookup = defaultdict(list)
     for annotation in annotations:
         annotation_lookup[getattr(annotation, 'image_id', None) or
-                          getattr(annotation, 'name', None)].append(annotation)
-    return annotation_lookup
+                          getattr(annotation, 'name')].append(annotation)
+    return annotation_lookup 
```

Comment:
what does removing `None` exactly fix and how?

Reply:
Im honestly not sure it was how the original author had it. I added the None in as a default value because some of our label.annotations type don't have a name resulting in an error. But that broke a test. If we wanted it back I can look at the test

Reply:
I figured it out the dictionary key was resolving weird since it could not pick up a value in the or state. I added logic to give a default value if name or image is not available and the test for python 3.8 works